.flake8

```diff
@@ -0,0 +1,31 @@
+[flake8]
+
+exclude = 
+    ./freemocap/data_layer/export_data/blender_stuff/blender_bpy_export_scripts/*
+    ./freemocap/core_processes/capture_volume_calibration/anipose_camera_calibration/anipose_camera_calibrator.py
+    ./freemocap/core_processes/capture_volume_calibration/anipose_camera_calibration/freemocap_anipose.py
+    ./.nox
+
+per-file-ignores = 
+    ./freemocap/core_processes/process_motion_capture_videos/process_recording_folder.py:C901
+    ./freemocap/core_processes/detecting_things_in_2d_images/mediapipe_stuff/mediapipe_skeleton_detector.py:C901
+    ./freemocap/core_processes/detecting_things_in_2d_images/mediapipe_stuff/convert_mediapipe_npy_to_csv.py:E203
+    ./experimental/batch_process/batch_process.py:E226
+# C901: function is too complex
+# E203: whitespace before ':' (PEP8 says this doesn't apply to slicing)
+# E226: missing whitespace around arithmetic operator (file slashes misread as arithmetic)
+
+max-line-length = 88
```

Comment:
I know that 88 is standard for `black`, but I was thinking about bumping this number to 100 or 120 in alignment with our "use long descriptive variable names" style/vibe ( which aligns with our unique-ish status as a student-focused technical scientific software). 

Thoughts?

Reply:
I think that's totally fair, and that seems to be the most commonly changed elements of Black.

Right now everything in this PR has been linted with Black with default parameters, so we can either revert that change and relint with a longer line length, or leave it linted as is but just be more permissive in the future of longer lines.